mpv

```diff
@@ -126,40 +128,64 @@ start_mpv(){
 		set -- "http://0.0.0.0:${random_port}"
 		trap "kill_jobs ; exit 0" INT HUP
 	fi
-	# set mpv+ytdl as fallback
 	# set delay to start activity first before clearing the temp file
-	if am start --user current -R 2 -a android.intent.action.VIEW -d "${1}" -n 'is.xyz.mpv/.MPVActivity' >/dev/null 2>&1; then
-		(sleep 5 && : > "${app_readable_file}") &
+	
+	case "${player}" in
+	    mpvkt)
+	        am start --user current -R 2 -a android.intent.action.VIEW -d "${1}" -t video/any -p 'live.mehiz.mpvkt' >/dev/null 2>&1
+	        xcode_mpv="$?"
+	        ;;
+	    mpvytdl)
+	        am start --user current -R 2 -a android.intent.action.VIEW -d "${1}" -n 'is.xyz.mpv.ytdl/is.xyz.mpv.MPVActivity' >/dev/null 2>&1
+	        xcode_mpv="$?"
+	        ;;
+	    default)
+	        am start --user current -R 2 -a android.intent.action.VIEW -d "${1}" -n 'is.xyz.mpv/.MPVActivity' >/dev/null 2>&1
+	        xcode_mpv="$?"
+	        ;;
+	    *)
+	        err "Exception: Invalid player"
+	        ;;
+	esac
+	if [ "${xcode_mpv}" = 0 ]; then
+	    (sleep 10 && : > "${app_readable_file}") &
```

Comment:
User might've not want to overwrite their current mpv config.
I'll fix this later